scripts/bootstrap.sh

```
@@ -16,6 +16,7 @@ MCP_METHOD="python"
 UV_FROM=""
 GRAPH_ENABLED=true  # bootstrap.sh では利便性のためデフォルトで有効（アプリデフォルトは false）
 POSTGRES_SSL=false
+CACHE_BACKEND=""
```

Shellcheck (0.11.0)

[warning] CACHE_BACKEND appears unused. Verify use (or export if used externally).

(SC2034)